src/core/services/compiler/compiler.js

```diff
@@ -10,74 +10,90 @@ angular
 
 /**
  * @ngdoc service
- * @name $mdCompiler
+ * @name $mdCompilerProvider
  * @module material.core.compiler
  * @description
- * The $mdCompiler service is an abstraction of AngularJS's compiler, that allows developers
- * to easily compile an element with options like in a Directive Definition Object.
+ * The `$mdCompiler` is able to respect the AngularJS `$compiler.preAssignBindingsEnabled` state when using
```

Comment:
`$compiler.preAssignBindingsEnabled` --> $compileProvider.preAssignBindingsEnabled` (here and elsewhere).